task_manager/src/lib.rs

```diff
@@ -144,6 +146,15 @@ pub struct TaskManagerOpts {
     pub max_db_size: usize,
 }
 
+#[derive(Debug, Serialize, ToSchema)]
+pub struct TaskReport {
```

Comment:
I think it can be combined into above TaskDescriptor
```
pub struct TaskDescriptor {
    pub chain_id: ChainId,
    pub blockhash: B256,
    pub proof_system: ProofType,
    pub prover: String,
}
```
I missed the prove_type in TaskDescriptor before.

Reply:
I'll combine it